README.md

```diff
@@ -48,6 +48,67 @@ module.exports = function (grunt) {
 };
 ```
 
+## Customizing configurations
```

Comment:
This should really be a subsection of the "Configuring Individual Checks" section.  That would also ensure that "how to run this" comes before "how to configure this", which seems like the right order.